charts/quickwit/values.yaml

```diff
@@ -313,50 +349,33 @@ bootstrap:
 
 
 # Quickwit configuration
+# Warning: This config is writed directly into a configMap
+# to avoid passing sensitive value you can pass environment variables.
+# https://quickwit.io/docs/configuration/node-config#using-environment-variables-in-the-configuration
 config:
-  # Metastore configuration.
-  # Default to `qwdata/indexes` if not set.
-  # S3 metastore
-  # metastore_uri: s3://quickwit/indexes
-  #
-  # Postgres metastore
+  version: 0.8
+  listen_address: 0.0.0.0
+  gossip_listen_port: 7282
+  data_dir: /quickwit/qwdata
+  default_index_root_uri: s3://quickwit/indexes
+
   # postgres:
-  #   host: ""
-  #   port: 5432
-  #   database: metastore
-  #   username: quickwit
-  #   password: ""
   #   max_num_connections: 50
 
-  # Storage configuration.
   # storage:
     # s3:
       # endpoint: "http://custom-s3-endpoint"
       # region: eu-east-1
       # We recommend using IAM roles and permissions to access Amazon S3 resources,
       # but you can specify a pair of access and secret keys if necessary.
       # access_key_id: <my access key>
-      # secret_access_key: <my secret key>
+      # secret_access_key: ${AWS_ACCESS_KEY_ID}
     # azure:
-      # account: "my-azure-blob-account-name"
-      # access_key: "my-azure-blob-access-key"
-
-  default_index_root_uri: s3://quickwit/indexes
-
-  # Indexer settings
-  # indexer:
-  #   split_store_max_num_bytes: 200G
-  #   split_store_max_num_splits: 10000
-  # Ingest API settings
-  # ingest_api:
-  #   max_queue_memory_usage: 2GiB
-  #   max_queue_disk_usage: 4GiB
-  # Searcher settings
-  # searcher:
-  #   fast_field_cache_capacity: 10G
-  #   split_footer_cache_capacity: 1G
-  #   max_num_concurrent_split_streams: 100
```

Comment:
What was the rational behind removing these? I find its a good example of things we can configure here.

Reply:
Old refactor that did not support this configuration. I will add it back!